### ISSUE TITLE
Can't load files with non-english letters in filename or path on windows

### ISSUE THREAD

**Issue by SkeLLLa** [nicfit/eyeD3#146]
eyeD3 cant load files containing any non-english letters like `ü` or `и` on windows. On linux it works well.

**Comment by 1deterministic** [comment]
Same problem here :/

**Comment by SkeLLLa** [comment]
@1Deterministic I've created simple workaround for this issue by creating temporary file (e.g. whit guid instead of real name)

**Comment by 1deterministic** [comment]
Wow that makes sense! I spent the day trying to fix that, but I was looking inside the eyeD3 code actually. I was able to open a file from a folder named with non-english letters ("á" was its name) under Windows by changing the line `return self.from_file(filename)` in the file `eyed3/utils/__init__.py ` ([this one](https://github.com/nicfit/eyeD3/blob/master/src/eyed3/utils/__init__.py)) to `return self.from_buffer(filename)` but it was pure trial and error with [this question from stackoverflow](https://stackoverflow.com/questions/34836792/python-magic-cant-identify-unicode-filename) and I have no experience in Python and I have no idea of the possible implications of using something like that. Maybe this info will help the maintainer to find the problem without spending all the day beating his head on the table the way I was haha
